colbert/training/training.py

```diff
@@ -45,16 +45,18 @@ def train(config: ColBERTConfig, triples, queries=None, collection=None):
         raise NotImplementedError()
 
     if not config.reranker:
-        colbert = ColBERT(name=config.checkpoint, colbert_config=config)
+        colbert = ColBERT(name=config.checkpoint, colbert_config=config, device_type=DEVICE.type)
     else:
         colbert = ElectraReranker.from_pretrained(config.checkpoint)
+        colbert.to(DEVICE)
 
-    colbert = colbert.to(DEVICE)
```

Comment:
was this doing anything earlier for ColBERT given there was a `self.model.to(DEVICE)` inside `BaseColBERT`?